Std/Data/Rat/Basic.lean

```diff
@@ -108,7 +108,7 @@ protected def ofScientific (m : Nat) (s : Bool) (e : Nat) : Rat :=
   else
     (m * 10 ^ e : Nat)
 
-instance : OfScientific Rat where ofScientific := Rat.ofScientific
+-- Instance of `OfScientific Rat` is defined via `RatCast` in `Std.Data.Classes.RatCast`.
```

Comment:
I don't think this is actually true, because there is no `RatCast Rat` instance in the other file.

Reply:
how about something like "Instance of `OfScientific Rat` is to be obtained via `RatCast`. See `Std.Data.Classes.RatCast`"? just thought it would be helpful to have some documented pointer to there from here, one way or another

Reply:
Also, *should* there be an instance of `RatCast Rat` in std? Currently you need mathlib4 to get one, I believe

Reply:
> Also, _should_ there be an instance of `RatCast Rat` in std? Currently you need mathlib4 to get one, I believe

I think `RatCast Rat` belongs in std4.  We also have `NatCast Nat` and `IntCast Int` here.

Reply:
I actually can't find `NatCast Nat` or `IntCast Int` in std4...were they removed?

Reply:
Ah, we have `NatCast Int` though.  I think the missing "reflexive" instances are just an oversight.

Reply:
Oh, okay! Shall I add them all in, then?